BGL/include/CGAL/boost/graph/Face_filtered_graph.h

```diff
@@ -104,6 +104,43 @@ struct Face_filtered_graph
 
   typedef Face_filtered_graph<Graph, FIMap, VIMap, HIMap> Self;
 
+  /*!
+   * \brief constructs an empty face filtered graph (no face is selected)
+   *
+   * \tparam NamedParameters a sequence of named parameters
+   *
+   * \param graph the underlying graph.
+   *
+   * \param np optional sequence of named parameters among the ones listed below
+   *
+   * \cgalNamedParamsBegin
+   *   \cgalParamBegin{face_index_map}
```

Comment:
Shouldn't this be `\cgalParamNBegin` ?
Might also solve the `TK_COMMAND_BS ` problem (i.e. it found a `@` or `\`).
Looks like the counting is slightly wrong.

Reply:
The branch is indeed outdated.